modules/openapi-generator/src/main/resources/typescript/package.mustache

```diff
@@ -69,7 +69,7 @@
     "rxjs": "^6.4.0",
     {{/useRxJS}}
     {{#useInversify}}
-    "inversify": "^5.0.1",
+    "inversify": "^6.0.1",
```

Comment:
Inversify 5.x doesn't work properly with our current TS version. It's this issue here: https://stackoverflow.com/questions/75776252/inversify-typescript-5-ts1239-unable-to-resolve-signature-of-parameter-decor